### PR TITLE
Sanity check

### PR DESCRIPTION
**Pull request by dantpm**
Add Signed-off-by: line.

All issues addressed in the old pull request #775.  If not, please respond there.

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -11,16 +11,20 @@ following sections describe them for the supported platforms.
...
-** OpenSSL
+
+   OpenSSL libraries and header files
+
 * Unit test suite (see ./configure option --enable-unit):
-** cmocka unit test framework
+
+   cmocka unit test framework
+
 Most users will not need to install these dependencies.
```
Comment (AndreasFuchsSIT) [inline INSTALL.md]: All changes in this file are not part of the sanity-checking.
Please put them into a separate commit.

**Comment by dantpm** [comment]
I am still in merge hell.
Weirdly there are now commits showing up in this branch that are already present in upstream/master:
https://github.com/tpm2-software/tpm2-tss/pull/792/commits

**Comment by martinezjavier** [comment]
@dantpm I think is due your merge commit https://github.com/tpm2-software/tpm2-tss/pull/792/commits/3b439300314fa64c808a0d7063940d55b789dab5.

So I think you should start from a fresh branch, something like:

```
$ git branch -m sanity-check-old
$ git checkout -b sanity-check origin/master
$ git cherry-pick $SHA-1 # this is the commit that you want to add

Do any changes that you need for that commit

$ git commit -a --amend
$ git push dantpm sanity-check -f
$ git branch -D sanity-check-old
```

**Comment by dantpm** [comment]
I will try that this morning and see if it works. Thanks again for all the tips!
I have no problem using git now except for the "rebase" operation, which is tedious and error prone.
Changes I never made show up as merge conflicts and commits already in upstream master show up in the list of commits.  But I will try the above.

**Comment by dantpm** [comment]
git rebase fails to merge my commits.
Also git show does not show my commit, but shows the most recent commit (which I pulled from master)
When I run git rebase -i
and edit the .git/rebase-merge/git-rebase-todo file to squash my older commits, I get an error:
error: could not apply cf79d0dc4678695e90a1c256196730ca700aab1b... script: Add Sanity Checks to int-log-compiler.sh
